frontend/ui/data/keyboardlayouts/ja_keyboard.lua

```diff
@@ -1,103 +1,106 @@
 return {
     min_layer = 1,
-    max_layer = 12,
+    max_layer = 4,
     shiftmode_keys = {[""] = true},
     symbolmode_keys = {["記号"] = true, ["かな"] = true},
-    utf8mode_keys = {["゛"] = true},
+    utf8mode_keys = {["🌐"] = true},
     umlautmode_keys = {["゜"] = true},
     keys = {
         -- first row
-        {  --  1           2       3       4       5        6       7       8       9        10      11      12
-            { "ア",        "あ",    "~",    "`",    "ア",    "あ",    "~",    "`",    "ア",    "あ",    "~",    "`", },
-            { "カ",        "か",    "!",    "1",    "ガ",    "が",    "!",    "1",    "カ",    "か",    "!",    "1", },
-            { "サ",        "さ",    "@",    "2",    "ザ",    "ざ",    "@",    "2",    "サ",    "さ",    "@",    "2", },
-            { "タ",        "た",    "#",    "3",    "ダ",    "だ",    "#",    "3",    "タ",    "た",    "#",    "3", },
-            { "ナ",        "な",    "$",    "4",    "ナ",    "な",    "$",    "4",    "ナ",    "な",    "$",    "4", },
-            { "ハ",        "は",    "%",    "5",    "バ",    "ば",    "%",    "5",    "パ",    "ぱ",    "%",    "5", },
-            { "マ",        "ま",    "^",    "6",    "マ",    "ま",     "^",    "6",    "マ",    "ま",    "^",    "6", },
-            { "ヤ",        "や",    "&",    "7",   "ヤ",     "や",    "&",    "7",    "ヤ",    "や",    "&",    "7", },
-            { "ラ",        "ら",    "*",    "8",    "ラ",     "ら",    "*",    "8",    "ラ",    "ら",    "*",    "8", },
-            { "ワ",        "わ",    "(",    "9",    "ワ",     "わ",    "(",    "9",    "ワ",    "わ",    "(",    "9", },
-            { "ァ",        "ぁ",    ")",    "0",    "ァ",     "ぁ",    ")",    "0",    "ァ",    "ぁ",    ")",    "0", },
-            { "ャ",        "ゃ",    "_",    "-",    "ャ",     "ゃ",    "_",    "-",    "ャ",    "ゃ",    "_",    "-", },
-            { "゛",         "゛",     "+",    "=",    "゛",      "゛",     "+",    "=",    "゛",     "゛",     "+",    "=", },
+        {  --  1       2       3       4
+            { "ア",    "あ",    "~",    "`", },
+            { "カ",    "か",    "!",    "1", },
+            { "サ",    "さ",    "@",    "2", },
+            { "タ",    "た",    "#",    "3", },
+            { "ナ",    "な",    "$",    "4", },
+            { "ハ",    "は",    "%",    "5", },
```

Comment:
Any idea why that one (2nd char among the 4) is highlighted by github diff?
(And actually always one at the same place in the 3 following sections.)

Reply:
I double checked the characters and they seem to be exactly the same, so I can only assume it's an obscure GH bug.

![Screenshot_20191118_232622](https://user-images.githubusercontent.com/202757/69099385-ebf11080-0a5a-11ea-8871-3a9d4dc49159.png)

Reply:
@poire-z I now understand why it is. It relates to why I accidentally deleted a few layers in the Japanese layout. The tenth layout is exactly the same as the second, except on this specific line where it's `ぱ` instead of `は`. But GH (intelligently?) regards this as a change rather than as deleting the third to tenth layout. It probably has something to do with the number of spaces.

![Screenshot_2019-12-30_16-26-15](https://user-images.githubusercontent.com/202757/71588284-36b57c00-2b21-11ea-90ee-e4dbf3517cfc.png)